packages/terra-alert/src/Alert.jsx

```diff
@@ -194,6 +203,15 @@ const Alert = ({
     </div>
   );
 
+  useEffect(() => {
+    const isAlert = role === 'alert' || defaultRole === 'alert';
+    if (!disableAlertActionFocus && isAlert && action && alertRef?.current?.focus) {
+      alertRef.current.focus();
+    }
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [alertRef.current]);
```

Comment:
It looks like changing the [current property of the ref doesn't trigger a re-render](https://github.com/facebook/react/issues/14387#issuecomment-503616820) meaning this useEffect wouldn't get called, so is this just happening once after mounting when the ref is initially set up?

Reply:
Oh yeah that's a good point, not sure why I even added the alertRef cause it shouldn't even change actually. But yeah, we are just wanting this to run after the notification banner is mounted.

Not sure if there is a preference but exhausting the deps in the useEffect could potentially cause a re-render and it could refocus unexpectedly, so for now I just removed all of them to just have the focus occur on initial mount

Updated here https://github.com/cerner/terra-core/pull/3797/commits/292fc6406dbbe795bc23956ea70940228d723749